docs/platforms/javascript/common/enriching-events/scopes/index.mdx

```diff
@@ -143,3 +143,11 @@ In the following example we use <PlatformIdentifier name="with-scope" /> to atta
 The scope inside the `withScope()` callback is only valid inside of the callback. Once the callback ends, the scope will be removed and no longer applied. The inner scope is only applied to events that are captured inside of the callback. `withScope()` will clone (or fork) the current scope, so that the current scope is not modified. This allows you to
 more easily isolate pieces of context information to specific locations in your code or
 even call <PlatformIdentifier name="clear" /> to briefly remove all context information.
+
+<PlatformCategorySection supported={['server']}>
+## Using `withIsolationScope`
+
+`withIsolationScope` works fundamentally the same as `withScope`, but it will fork the isolation scope instead of the current scope. Generally, the isolation scope is meant to be forked less frequently than the current scope, and in most cases the SDK will handle this automatically for you. But in cases where you e.g. want to capture SDK events in a middleware (which happens before the request is processed and thus before the SDKs automatic handling), or if you want to isolate a non-request process (e.g. a background job), you can use `withIsolationScope` to create a new isolation scope that is only active for the duration of the callback:
+
+<PlatformContent includePath="enriching-events/scopes/with-isolation-scope" />
+</PlatformCategorySection>
```

Comment:
```suggestion
`withIsolationScope` works fundamentally the same as `withScope` with the exception that it forks the isolation scope instead of the current scope. Generally, the isolation scope is meant to be forked less frequently than the current scope, and in most cases the SDK will handle this automatically for you. 

But in cases where you for example, want to capture SDK events in a middleware (which happens before the request is processed and thus before the SDK can automatically handle it), or if you want to isolate a non-request process (for example, a background job), you can use `withIsolationScope` to create a new isolation scope that's only active for the duration of the callback by instrumenting the below:

<PlatformContent includePath="enriching-events/scopes/with-isolation-scope" />
</PlatformCategorySection>
```

Reply:
I removed the part about middlewares because it turns out it is not actually correct. rewrote it slightly!